### PR TITLE
Amls 5817 - CYA and other accessibility changes

### PR DESCRIPTION
**Pull request by matthewharris1**
Aria described by changes
CYA bullet changes
Hint text change
## How Has This Been Tested?
Unit
Manual

## Checklist

- [ ] Breaking change (fix or feature that would cause existing functionality to change).
- [x] Build passing.
- [x] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [ ] Requires acceptance test run.
- [x] Appropriate labels added.
- [x] RELEASE NOTES ADDED.

### CODE REVIEW

File: app/utils/CheckYourAnswersHelper.scala [hunk 1]
```diff
@@ -48,11 +48,21 @@ class CheckYourAnswersHelper(userAnswers: UserAnswers)(implicit messages: Messag
...
-        routes.TypeOfParticipantController.onPageLoad(CheckMode).url
-      )
+      if(x.size > 1) {
+        AnswerRow(
+          HtmlFormat.escape(messages("typeOfParticipant.checkYourAnswersLabel")),
+          typeOfParticipantHtml(x),
+          routes.TypeOfParticipantController.onPageLoad(CheckMode).url
+        )
+      } else {
+        val v = x.map(value => value.toString).head
```
Comment (ajchilts) [inline app/utils/CheckYourAnswersHelper.scala:58]: I know I provided this example but I think we should change 'v' to something more meaningful?

File: app/views/ViewUtils.scala [hunk 2]
```diff
@@ -24,4 +24,16 @@ object ViewUtils {
...
+    "value-yes"
+  }
+
+  def percentageExpectedTurnoverErrorId = {
+    "percentageExpectedTurnover.zeroToTwenty"
+  }
+
+  def genericCheckboxErrorId = {
+    "value_0"
+  }
```
Comment (ajchilts) [inline app/views/ViewUtils.scala:38]: I know I provided these examples however; I did not give a comprehensive example. Are we sure this covers every page on which the error occurs?

File: release_notes/AMLS-5835.txt [hunk 3]
```diff
@@ -1 +0,0 @@
- + [AMLS-5835] - 'Updated links that look like buttons to properly mimic the behaviour of a button element'
```
Comment (ajchilts) [inline release_notes/AMLS-5835.txt:1]: This is strange... Where is the release note for this change?

File: app/views/ViewUtils.scala (hunk 2 above)
Comment (matthewharris1) [reply app/views/ViewUtils.scala:38]: It wasn't implemented on the LinkedTransactions page. It is now :)

File: release_notes/AMLS-5835.txt (hunk 3 above)
Comment (matthewharris1) [reply release_notes/AMLS-5835.txt]: Re-added this

File: app/utils/CheckYourAnswersHelper.scala (hunk 1 above)
Comment (matthewharris1) [reply app/utils/CheckYourAnswersHelper.scala]: Done

**Pull request merged**